terraform/modules/cnames/main.tf

```
@@ -0,0 +1,15 @@
+locals {
+  type      = "CNAME"
+  match     = regex("(^[^.]+)(\\..*)", var.quilt_web_host)
```

```suggestion
  match     = regex("^([^.]+)(\\..*)$", var.quilt_web_host)
```

Add a `$` at the end just to be safe, and move `^` out of the capture group, to make it easier to read?